modules/Microsoft.Network/dnsResolvers/.test/common/deploy.test.bicep

```diff
@@ -64,5 +64,14 @@ module testDeployment '../../deploy.bicep' = {
       Environment: 'Non-Prod'
       Role: 'DeploymentValidation'
     }
+    roleAssignments: [
+      {
+        roleDefinitionIdOrName: 'Reader'
+        description: 'Reader Role Assignment'
+        principalIds: [
+          'c99b4f9a-4268-4ab0-bd02-85d160b29a36' // carml-contributor-group
```

Comment:
This should/must be the ID of an MSI that is deployed in the dependencies.

Reply:
True